posts/2021/12/conda-recipe-grayskull.md

```diff
@@ -0,0 +1,124 @@
+<!--
+.. title: On Software Packages, Conda and Grayskull
```

Comment:
```suggestion
.. title: Conda and Grayskull, the Masters of Packaging 
```

please feel free to reject this suggestion, but i thought it would fun to riff off of he-man, masters of the universe

Reply:
Thanks, @tonyfast. I think it's a good idea.
I'm making it: "Conda and Grayskull, the Masters of Software Packaging"